1-callbacks/optional.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+const validateUser = require("./validate-userOptional");
+
+const evaluateDataTerminal = () => {
+  if (process.argv.length !== 2) {
+    return process.argv.slice(2);
```

Comment:
(process.argv.length != 2 )
With this, the data that was entered through the terminal is known, normally process.argv.length == 2 when no arguments are received. if it is written:
node solution.js Danny Gabriel Jose Juan.
It would already have a size other than 2, it would have a size of 6; That is why this conditional is used that indicates that there is data entered manually